tomcat/README.md

```diff
@@ -52,9 +52,44 @@ Docker Hub.
     docker run --rm -it --hostname my-tomcat --name some-tomcat -p 8080:8080 clearlinux/tomcat
     ```
     
+
 <!-- Optional -->
 ### Deploy with Kubernetes
 
+This image can also be deployed on a Kubernetes cluster, such as [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).The following example YAML files are provided in the repository as reference for Kubernetes deployment:
+
+- [`tomcat-deployment.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/tomcat/tomcat-deployment.yaml): example using default configuration to create a basic tomcat service.
+
+- [`tomcat-deployment-index-custom.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/tomcat/tomcat-deployment-index-custom.yaml): example using your own custom index html to create a tomcat service.
+
+- [`tomcat-deployment-conf-custom.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/tomcat/tomcat-deployment-conf-custom.yaml): example using your own custom configuration to create a tomcat service.
+
+  
+
+Steps to deploy tomcat on a Kubernetes cluster:
+
+1. If you want to deploy `tomcat-deployment.yaml`
+
+   ```
+   kubectl create -f tomcat-deployment.yaml
+   ```
+
+   Or if you want to deploy `tomcat-deployment-index-custom.yaml`
+
+   ```
+   kubectl create -f tomcat-deployment-index-custom.yaml
+   ```
+
+   Or if you want to deploy `tomcat-deployment-conf-custom.yaml`
+
+   ```
+   kubectl create -f tomcat-deployment-conf-custom.yaml
+   ```
+
+2. Navigate to [http://\<nodeIP\>:30001](http://\<nodeIP\>:30001) in your browser. , where 30001 is the port number defined in your service.
```

Comment:
"browser. ,", remove the full stop here?

Reply:
> "browser. ,", remove the full stop here?

My miss!